DECLARING_DEPENDENCIES.md

```diff
@@ -0,0 +1,209 @@
+version of the google-cloud-storage artifact is omitted.
+
+See the "Choosing BOMs" section below if you need to import more than one BOM.
+
+### Gradle
+
+Gradle automatically selects the highest version in the dependency
+tree of any given dependency.
+
+#### Import BOMs to ensure your dependencies are consistent
+
+It is possible to accidentally use versions of different artifacts
+from a library from different releases and experience a dependency
+conflict. To fix this problem, you can import a BOM in Gradle to force
+consistent versions, as long as you are using at least Gradle 4.6.
+
+In order to make use of a BOM, do the following:
+
+- Gradle 4.x (4.6+):
+  - Add the following to your `settings.gradle`: `enableFeaturePreview('IMPROVED_POM_SUPPORT')`
+- Add a dependency on the BOM for the library you depend on
+- Remove the version from the dependency declarations of the artifacts in that library
+
+For an example, see [gax-java#690](https://github.com/googleapis/gax-java/pull/690/files).
+
+See the "Choosing BOMs" section below if you need to import more than one BOM.
+
+## Choosing BOMs
+
+Generally, you should import the highest-level BOM that your project
+depends on. For example, if you depend on both `grpc-java` and
+`google-cloud-java`, then use `google-cloud-bom` (since `google-cloud-java`
+depends on `grpc-java`). This should get you the versions of everything
+you need, because the transitive dependencies of that library should
+all be compatible as well.
+
+However, if your project independently pulls in transitive
+dependencies at different versions, you may also need to specify
+lower-level BOMs to ensure compatibility, since each BOM only controls
+the versions of the library that generates it. Since the GCP Java
+libraries follow semver, you should be able to pick the highest
+version in your dependency tree for any particular library. Here is
+the process to follow for each library that has BOM support:
+
+1. Identify the highest version seen for any artifact produced by that library
+  a. For example, if you see `gax:1.34.0` and `gax-grpc:1.42.0`, then use version 1.42.0
+2. Import the BOM corresponding to that library using the highest version you see
+  a. For the example above, use `gax-bom:1.42.0`
+3. Repeat as long as there are other libraries with BOM support where
+   you see multiple versions of the library's artifacts
+
+Note that there will not necessarily be a BOM available at the version
+you need. See the Library reference section below for the first
+available version of each BOM.
+
+## BOM reference
+
+None of the GCP open source Java libraries were initially released
+with BOMs - they were added later to provide better consistency for
+users. The following table shows the first version that each library
+published a BOM for, and the BOM artifact name.
+
+| library | BOM artifact | First available version |
+| --- | --- | --- |
+| guava | com.google.guava:guava-bom | 27.1-jre/27.1-android (8 Mar 2019) |
```

Comment:
```suggestion
| guava | com.google.guava:guava-bom | 27.1-jre/27.1-android (08-Mar-2019) |
```

Just for consistency. BTW, U.S. English uses "month-day-year" order somehow, and Google uses U.S. English.

Reply:
I copied them out of search.maven.com, which is why they have this format. 

Speakers of U.S. English can understand this format when the months are spelled out :-)